src/components/SpaceSidebarFooter.vue

```diff
@@ -1,40 +1,30 @@
 <script setup lang="ts">
-import { computed } from 'vue';
+const props = defineProps<{ space?: Record<string, any> }>();
```

Comment:
Why is space optional now? If it is because of some loading order, this component should maybe not be displayed at all as long as space is not loaded. If it stays like that, `socials` should be computed then.

Reply:
The component should always be loaded as we always show the icons, no matter if the space has these socials set.